src/modules/aws_ecr/index.ts

```diff
@@ -115,13 +113,13 @@ class RepositoryImageMapper extends MapperBase<RepositoryImage> {
       repository.repositoryName,
       repository.registryId,
     );
-    if (images && images.imageIds && images.imageIds.length > 0) {
-      await this.deleteRepositoryImage(
-        client,
-        images.imageIds,
-        repository.repositoryName,
-        repository.registryId,
-      );
+    if (images && images.imageDetails) {
+      for (const image of images.imageDetails) {
+        if (image.imageDigest && image.imageTags && image.imageTags.length > 0) {
```

Comment:
So the `image.imageTags` should always be set here? Is that a must? I see in the `repositoryImageMapper` it's being treated as possibly-undefined. But here we're relying on its existence. Just a non-blocking question.